src/applications/resources-and-support/style.scss

```diff
@@ -0,0 +1,22 @@
+.medium-screen-va-border-left-radius--0 {
+  @media (min-width: 768px) {
+    border-bottom-left-radius: 0;
+    border-top-left-radius: 0;
+  }
+}
+
+.medium-screen-va-background-color--white {
+  @media (min-width: 768px) {
+    background-color: #ffffff !important;
+  }
+}
```

Comment:
We have some sass [breakpoints](https://design.va.gov/design/breakpoints) we can use here instead of the 768s

Reply:
I tried using $medium-screen and it actually threw an error on me

Reply:
Oh yeah you gotta add `@import "~@department-of-veterans-affairs/formation/sass/shared-variables";`

Reply:
ah perfect! updated! thanks!